examples/marketing/giftcertificates.rb

```diff
@@ -0,0 +1,43 @@
+require 'bigcommerce'
+
+Bigcommerce.configure do |config|
+  config.auth = 'legacy'
+  # You will get this url when registering for an API key
+  config.url = ENV.fetch['BC_STORE_HASH']
+  config.username = ENV.fetch['BC_CLIENT_ID']
+  config.api_key = ENV.fetch['BC_ACCESS_TOKEN']
+end
+
+puts Bigcommerce::System.time
+puts Bigcommerce.api_limit
+
+# List gift certificates
+@gift_certificates = Bigcommerce::Giftcertificates.all
```

Comment:
Convention for ruby is generally to name multi-word classes as
`GiftCertificates` a la init-cap camelcase. This changes the file name to `gift_certificates`.